docs/using.md

```diff
@@ -1,166 +1,428 @@
+
+
+<a id="orgc7904ae"></a>
+
+## What is the system under test?
+
+Genny usually expects to be given a connection string to a system under test. While there's nothing preventing a user from writing a "workload" that spawns the system under test, there is not first-class support for such behavior. Genny is explicitly <span class="underline">not</span> a test orchestrator. For test orchestration needs, consider using [DSI](https://github.com/10gen/dsi/).
+
+By default, Genny will try to connect to a MongoDB server at `localhost:27017`. To pass a MongoDB connection string to Genny, use the `-u` option:
+
+    ./run-genny workload -u [arbitrary_URI] src/workloads/docs/InsertWithNop.yml
+
+For more details on how Genny handles connections, see [Connecting to the Server](#orgd6b0450).
+
+
+<a id="org3610c67"></a>
+
+## What is a workload?
+
+A **workload** is a repeatable procedure that Genny uses to generate load against a system under test. Genny workloads are written in yaml configs that describe how **actors** move through **phases**. This section describes each of these.
+
+
+<a id="orgdecc7ae"></a>
+
+### How are workloads configured?
+
+Here is an example of a simple Genny workload config:
+
+    SchemaVersion: 2018-07-01
+    Owner: "@10gen/dev-prod-tips"
+    Description: |
+      This is an introductory workload that shows how to write a workload in Genny.
+      This workload writes a few messages to the screen.
+    
+    Keywords:
+    - docs
+    - HelloWorld
+    
+    Actors:
+    - Name: HelloWorldExample
+      Type: HelloWorld
+      Threads: 2
+      Phases:
+      - Message: Hello Phase 0 🐳
+        Duration: 50 milliseconds
+      - Message: Hello Phase 1 👬
+        Repeat: 100
+
+Everything under the `Actor` key (where the magic happens) will be explained in the next section. First let's look at the other **required** keys:
+
+-   `SchemaVersion` - This is a basic versioning system used for Genny workload syntax. For the moment, any new workloads should have the value `2018-07-01` for this key.
+-   `Owner` - This should have an identifier for the team that owns the workload, ideally an @-mentionable GitHub team.
+-   `Description` - This should contain a written description of the workload. It's recommended to go into as much detail as possible, since understanding the performance issue behind why a workload was written can be difficult months or years later.
+-   `Keywords` - These should be searchable keywords associated with your workload. Include keywords for actors used, operations performed, qualities of the system under test that are expected, etc.
+
+Workload configurations can be found in [./src/workloads](../src/workloads) from the Genny repo root. Organization of this directory is arbitrary as far as Genny is concerned, though example workloads should be in the `docs` subdir.
+
+
+<a id="org51d4d33"></a>
+
+### What is an actor?
+
```

Comment:
Any best practice recommendations for configuring actors ? Or anti patterns that need to be called out ?

Reply:
Added a few!